test/clt-tests/core/call-autocomplete-force-bigrams-vertical.rec

```diff
@@ -0,0 +1,179 @@
+––– comment –––
+Test for force_bigrams option in CALL AUTOCOMPLETE, CALL SUGGEST, and fuzzy search
+Issue: https://github.com/manticoresoftware/manticoresearch/issues/3853
+
+This test verifies that force_bigrams parameter works correctly in three contexts:
+1. CALL AUTOCOMPLETE (tests 1-9): autocomplete with bigram-based fuzzy matching
+2. CALL SUGGEST (tests 10-14): suggestions with bigram-based fuzzy matching
+3. SELECT with OPTION fuzzy=1 (tests 15-19): regular search with fuzzy option
+
+The force_bigrams option helps find transposition errors (e.g., "ipohne" -> "iphone")
+in longer words where bigrams are not used by default.
+––– block: ../base/start-searchd-with-buddy –––
+––– input –––
+mysql -P9306 -h0 -e "drop table if exists t"
+––– output –––
+––– input –––
+mysql -P9306 -h0 -e "create table t(f text) min_infix_len='2'"
+––– output –––
+––– input –––
+mysql -P9306 -h0 -e "insert into t values(1,'iphone developer george something planet letter')"
+––– output –––
+––– comment –––
+Test 1: force_bigrams=1 finds transposition errors
+––– input –––
+mysql -P9306 -h0 -e "call autocomplete('ipohne', 't', 1 as force_bigrams)\G"
+––– output –––
+*************************** 1. row ***************************
+query: iphone
+––– comment –––
+Test 2: force_bigrams=0 does NOT find transpositions in long words
+––– input –––
+mysql -P9306 -h0 -e "call autocomplete('ipohne', 't', 0 as force_bigrams)\G"
+––– output –––
+––– comment –––
+Test 3: Long word transposition + force_bigrams
+––– input –––
+mysql -P9306 -h0 -e "call autocomplete('develepor', 't', 1 as force_bigrams)\G"
```

Comment:
Why not use the same keyword as before? If you change both the query and force_birams at the same time, it’s hard to tell whether it’s actually working.